### DISCUSSION TITLE
About upgrading my create3 to turtlebot4 lite and learning the turtlebot4 documentation

### DISCUSSION THREAD

**Discussion by alain-lnpcrd** [iRobotEducation/create3_docs#550] [Show and tell]
Hello everybody,
Hello @slowrunner
My turtlebot4 admiring his future playground
![robot1](https://github.com/iRobotEducation/create3_docs/assets/74408456/c4e2984e-a1bb-4604-aac7-fad334357af7)
where it should never go.

Model defined with turtlebot4-setup
![Screenshot from 2024-02-18 13-44-56](https://github.com/iRobotEducation/create3_docs/assets/74408456/40b8281a-bd85-4d30-8b28-12f61be9aa86)

After generating the map with "ros2 launch tur looking to use lebot4_navigation slam.launch.py"
I'm trying to navigate my turtlebot4 in the map and I can't do it.
Nav2 / Launching navigation "ros2 launch turtlebot4_navigation nav_bringup.launch.py slam:=off localization:=true map:=/home/me/ma_maison.yaml"
![Capture d’écran du 2024-02-18 15-35-56](https://github.com/iRobotEducation/create3_docs/assets/74408456/281bdabc-1298-4c70-a4f8-96a4d0178f43)

and there I discovered that locating the robot solved the problem of the unknown "frame map" . (red stain in room on lower left upper wall)

My current problem is running the "Nav2 Goal" operation (in chapter 5. Navigation)
" initial pose first, then goal pose " like example to see in this paragraph.
Normally the path to execute should appear on the map but nothing happens despite multiple attempts.

**Comment by slowrunner** [comment]
Congratulations! That is great progress indeed - creating a map and localizing in the map.  That is as far as I succeeded in slam-toolbox.  My testing did seem to navigate if I set the goal close in front of the Create3, but not if there was an obstacle to avoid or if the goal was behind the robot's current heading.

**Comment by alain-lnpcrd** [comment]
Hello,In fact, to read only half of the information, I had simply forgotten an operation, launching " ros2 launch turtlebot4_navigation nav2.launch.py "
The movement takes place but often fails to reach its destination, and there the only solution found is to restart all the processes
